### ISSUE TITLE
AdaBatch: Adaptive Batch Sizes for Training Deep Neural Networks

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#549]
## 一言でいうと

学習率ではなく、バッチサイズを調整することで同等の結果が得られるという検証結果。実験では、バッチサイズをエポック数に応じて倍々にしていくことで精度が上がることを確認できた(ただ、学習率の低減を使用していないわけではない)。
"For the adaptive method we decay the learning rate by 0.75 and simultaneously double the batch size at the same 20-epoch intervals."

![image](https://user-images.githubusercontent.com/544269/33750685-05cde892-dc19-11e7-9ba8-60d1a8c616e5.png)

### 論文リンク

https://arxiv.org/abs/1712.02029

### 著者/所属機関

Aditya Devarakonda, Maxim Naumov, Michael Garland

* University of California
* NVIDIA

### 投稿日付(yyyy/MM/dd)

2017/12/6
